src/ShopifyClient.php

```diff
@@ -568,13 +568,18 @@ private function validateConnectionOptions(array $connectionOptions)
     /**
      * @return GuzzleClient
      */
-    private function createDefaultClient(): GuzzleClient
+    private function createDefaultClient(array $guzzleMiddleware = []): GuzzleClient
     {
         $baseUri = 'https://' . str_replace('.myshopify.com', '', $this->connectionOptions['shop']) . '.myshopify.com';
 
         $handlerStack = HandlerStack::create(new CurlHandler());
         $handlerStack->push(Middleware::retry([$this, 'retryDecider'], [$this, 'retryDelay']));
 
+        foreach ($guzzleMiddleware as $curMiddleware)
```

Comment:
Can you please move the opening { to follow the same code conventions ? :)

Reply:
Sure, sorry about that.